src/awkward/_nplikes/numpy.py

```diff
@@ -36,6 +36,10 @@ def char(self):
     def ndarray(self):
         return self._module.ndarray
 
+    @property
+    def atleast_1d(self):
+        return self._module.atleast_1d
```

Comment:
I think this should rather live on the [`ArrayModuleNumpyLike`](https://github.com/scikit-hep/awkward/blob/main/src/awkward/_nplikes/array_module.py#L41) as a method.